master/buildbot/db/changes.py

```diff
@@ -56,7 +56,7 @@ def thd(conn):
     def addChange(self, author=None, files=None, comments=None, is_dir=None,
                   revision=None, when_timestamp=None, branch=None,
                   category=None, revlink='', properties=None, repository='', codebase='',
-                  project='', uid=None):
+                  project='', uid=None, committer=None):
```

Comment:
I think for consistency let's put this the committer parameter just after author in all commits.

Reply:
Done